lib/tilex_web/models/developer.ex

```diff
@@ -6,14 +6,14 @@ defmodule Tilex.Developer do
   alias Tilex.{Developer, Post}
 
   schema "developers" do
-    field :email, :string
-    field :username, :string
-    field :google_id, :string
-    field :twitter_handle, :string
-    field :admin, :boolean
-    field :editor, :string
+    field(:email, :string)
```

Comment:
I'm ambivalent about this style, but it seems to be the convention for new Elixir projects. I also think its more approachable for people new to this or any programming language.